stf/model/prediction/creator.py

```diff
@@ -4,12 +4,17 @@
 
 from stf.model.prediction.xgboost.quantile import QuantileXGBPredictionModel
 from stf.model.prediction.xgboost.xgboost import XGBPredictionModel
+from stf.model.prediction.lightgbm.lightgbm import LGBPredictionModel
+
 
 
 class PredictionModelCreator:
 
     PROGNOSIS_MODEL_CONSTRUCTORS = {"xgb": XGBPredictionModel,
-                                    "xgb_quantile": QuantileXGBPredictionModel}
+                                    "xgb_quantile": QuantileXGBPredictionModel,
+                                    "lgb": LGBPredictionModel,
+                                    #"lgb_quantile": QuantileLGBPredictionModel,
+                                   }
```

Comment:
lgb_quantile is not yet implemented, please remove 'lgb_quantile' instead of commenting out